### PR TITLE
Do 1.0.1 release

### PR DESCRIPTION
**Pull request by kghbln**
- Add release notes for version 1.0.1
- Tweaked release notes to be consistent

### CODE REVIEW

**Comment by kghbln** [comment]
- [x] Update release notes - https://github.com/SemanticMediaWiki/SemanticCompoundQueries/pull/24
- [x] Update readme - https://github.com/SemanticMediaWiki/SemanticCompoundQueries/pull/24
- [x] Update version number (1.0.1) - https://github.com/SemanticMediaWiki/SemanticCompoundQueries/pull/24
  - [x] SemanticCompoundQueries.php
- [ ] Create tag
- [ ] Announce via mail
- [ ] Do a release tweet on Twitter

**Comment by kghbln** [comment]
@JeroenDeDauw It will be cool if you could merge with one commit.

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -12,7 +18,7 @@ Released on  September 21, 2016.
 * Added `compoundquery` API endpoint
 * Added `unsorted` parameter
 * Added composer support
-* General repository cleanup
+* Made general repository cleanup
```
Comment (JeroenDeDauw) [inline RELEASE-NOTES.md]: This is more consistent but the English is kinda weird

Comment (kghbln) [inline RELEASE-NOTES.md]: Well there is always room for improvement. That's a typo indeed.

**Pull request merged**